victron_ble/devices/base.py

```diff
@@ -261,6 +261,8 @@ class ACInState(Enum):
     0xA06F: "BlueSolar MPPT 150|45 rev2",
     0xA070: "BlueSolar MPPT 150|60 rev2",
     0xA071: "BlueSolar MPPT 150|70 rev2",
+    0xA0EC: "SmartLithium Battery 12V/160Ah",
+    0xA0EE: "SmartLithium Battery 24V/200Ah",
```

Comment:
```suggestion
    0xA0EE: "SmartLithium Battery 24V/200Ah",
    0xA0F0: "SmartLithium Battery 12V/330Ah",
```
Can add one more product code based on my installation.